src/main/java/org/sagebionetworks/web/client/widget/table/v2/TableEntityWidget.java

```diff
@@ -172,21 +200,41 @@ public void configure(EntityBundle bundle, Long versionNumber, boolean canEdit,
 		this.actionMenu = actionMenu;
 		this.entityTypeDisplay = EntityTypeUtils.getFriendlyEntityTypeName(bundle.getEntity());
 		reconfigureState();
+		showEditorIfEditableAndEmpty();
 	}
 
 	private void reconfigureState() {
+		initializeQuery();
+		if (this.hasQueryableData) {
+			initSimpleAdvancedQueryState();
+		}
 		configureActions();
-		checkState();
-		initSimpleAdvancedQueryState();

```

Comment:
- Renamed `checkState` -> `initializeQuery`.
- `initializeQuery` will set `this.hasQueryableData`.
- We call `configureActions` last now because it depends on `this.hasQueryableData`